signer/src/dkg/wsts.rs

```diff
@@ -0,0 +1,47 @@
+//! Contains WSTS helper types and impls for use within the DKG module.
+
+/// A helper enum to represent the different types of WSTS messages.
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
+pub enum WstsNetMessageType {
```

Comment:
`wsts::net::Message` is already an `enum`, so why is this necessary?

Reply:
`wsts::net::Message` variants all have inner values, was the issue. I can't e.g. say `.get_messages(wsts::net::Message::NonceRequest)`.